x/recovery/ibc_middleware.go

```diff
@@ -64,3 +64,11 @@ func (im IBCMiddleware) WriteAcknowledgement(
 ) error {
 	return im.keeper.WriteAcknowledgement(ctx, chanCap, packet, ack)
 }
+
+func (im IBCMiddleware) GetAppVersion(
+	ctx sdk.Context,
+	portID,
+	channelID string,
+) (string, bool) {
+	return "UnImplement", true
```

Comment:
same as above, missing godoc

Reply:
I don't understand why the string needs to be `"UnImplement"`. What does this mean?

Reply:
fixed